src/io/flutter/ProjectOpenActivity.java

```diff
@@ -40,12 +42,15 @@
 public class ProjectOpenActivity implements StartupActivity, DumbAware {
   public static final ProjectType FLUTTER_PROJECT_TYPE = new ProjectType("io.flutter");
   private static final Logger LOG = Logger.getInstance(ProjectOpenActivity.class);
+  private boolean indexingFinished = false;
 
   public ProjectOpenActivity() {
   }
 
   @Override
   public void runActivity(@NotNull Project project) {
+    project.getService(TimeTracker.class).setProjectOpenTime();
```

Comment:
We usually great a getService method on classes instead that passes in a Project instance. That is a bit cleaner than calling getService on the project itself.